npm-shrinkwrap.json

```diff
@@ -1,8 +1,5622 @@
 {
   "name": "@kontist/mock-solaris",
   "version": "1.0.38",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
```

Comment:
Why do we have these new 5000 lines of `packages` ? I don't think they should not be added.

Reply:
https://github.com/kontist/mock-solaris/pull/65
NPM 7 uses "lockfileVersion": 2 and the file is auto generated.
I've re-generated the file using npm install as it was outdated anyway: https://github.com/kontist/mock-solaris/pull/65/files